### PR TITLE
chore(ci): fix namespace cleanup

### PR DESCRIPTION
**Pull request by edvald**
Noticed some namespaces stacking up after CI runs. I think this should do the trick. @stefreak could you take a look?

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -219,15 +219,19 @@ commands:
...
-          name: Delete current namespace
-          command: kubectl get ns | grep ${CIRCLE_BUILD_NUM} | awk '{print $1}' | xargs -n 1 kubectl delete namespace --wait=false || true
+          name: Delete current namespace(s)
+          command: kubectl get ns | grep <<parameters.filter>> | awk '{print $1}' | xargs -n 1 kubectl delete namespace --wait=false || true
           when: always
       - run:
           name: Delete namespaces older than 1 hour
           # hack: find namespaces that contain '-ci' and their age ends with h or d, e.g. 1d5h or 24h
-          command: kubectl get ns | grep -E '(d|h)$' | grep -- '-ci' | awk '{print $1}' | xargs -n 1 kubectl delete namespace --wait=false || true
+          command: kubectl get ns | grep -E '(d|h)$' | grep -- '-testing' | awk '{print $1}' | xargs -n 1 kubectl delete namespace --wait=false || true
```
Comment (stefreak) [inline .circleci/config.yml:234]: do all the namespaces contain the string testing?

File: e2e/test/pre-release.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ describe("PreReleaseTests", () => {
     }
     // Override the userId variable
     if (process.env.CIRCLE_BUILD_NUM) {
-      command.push("--var", "userId=" + userId)
+      command.push("--var", "userId=" + userId + "-e2e")
```
Comment (stefreak) [inline e2e/test/pre-release.ts:55]: Is this change related to the cleanup? If so, I didn't understand the connection

Comment (edvald) [reply e2e/test/pre-release.ts:55]: The example projects all have a `userId` variable, which is interpolated into the k8s namespace name. This helps identify the namespace more precisely.

**Review by stefreak — APPROVED** [review]
👍  Nice
